app/models/invoice.rb

```diff
@@ -0,0 +1,36 @@
+# == Schema Information
+#
+# Table name: invoices
+#
+#  id              :integer          not null, primary key
+#  organization_id :integer          not null
+#  customer_id     :integer          not null
+#  starts_at       :datetime
+#  ends_at         :datetime         not null
+#  currency        :string           default("USD"), not null
+#  amount_cents    :integer          default(0), not null
+#  amount_currency :string           default("USD"), not null
+#  sent_at         :datetime
+#  paid_at         :datetime
+#  created_at      :datetime
+#  updated_at      :datetime
+#
+
+class Invoice < ActiveRecord::Base
+  monetize :amount_cents, with_model_currency: :currency
+
+  scope :ordered, -> { order('created_at DESC') }
```

Comment:
@yunixon order in model:
1. constants
2. associations
3. Fields declaration (monetize, attachments, attr_accessor, accepts_nested_attriutes)
4. validations
5. scopes
6. class methods
7. normal methods